### PR TITLE
Fix freaks details screen

### PR DESCRIPTION
**Pull request by srobert99**
#### Description
Fixed QA issues

#### [Task](task_url)
https://trello.com/c/zdOYA1fX/39-android-freaks-screen-details-screen

### CODE REVIEW

File: buildSrc/src/main/kotlin/BuildAndroidConfig.kt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 object BuildAndroidConfig {
-    const val APPLICATION_ID = "com.agilefreaks.freaks_catalog"
+    const val APPLICATION_ID = "com.agilefreaks.freaks.catalog"
```
Comment (cipry13) [inline buildSrc/src/main/kotlin/BuildAndroidConfig.kt]: wrong id (freaks_catalog)

**Comment by AlexandraDamaschin** [comment]
Please rebase this

**Comment by srobert99** [comment]
[Android: Freaks Screen - Details Screen](https://trello.com/c/zdOYA1fX/39-android-freaks-screen-details-screen)

File: features/freaks/src/main/res/layout/fragment_freak_details.xml [hunk 2]
```diff
@@ -78,7 +89,7 @@
                 android:layout_width="0dp"
                 android:layout_height="wrap_content"
                 android:layout_marginTop="@dimen/card_view_top_margin"
-                app:cardBackgroundColor="@color/variant_blue"
+                app:cardBackgroundColor="?attr/colorPrimary"
```
Comment (cipry13) [inline features/freaks/src/main/res/layout/fragment_freak_details.xml]: revert this change. color shouldn't be the primary blue.

File: features/freaks/src/main/res/layout/fragment_freak_details.xml [hunk 3]
```diff
@@ -143,8 +151,7 @@
                         android:layout_width="0dp"
                         android:layout_height="wrap_content"
                         android:layout_marginTop="@dimen/projects_level_top_margin"
-                        android:layout_marginBottom="@dimen/level_bot_margin"
-                        android:lines="1"
+                        android:paddingBottom="@dimen/level_bot_margin"
```
Comment (cipry13) [inline features/freaks/src/main/res/layout/fragment_freak_details.xml]: bot?

File: features/freaks/src/main/res/values-sw720dp/dimensions720dp.xml [hunk 4]
```diff
@@ -13,4 +13,7 @@
     <dimen name="card_view_top_margin">150dp</dimen>
     <dimen name="photo_margin_top">37dp</dimen>
     <dimen name="description_min_height">200dp</dimen>
+    <dimen name="image_rounded_background_stroke_width">6dp</dimen>
```
Comment (cipry13) [inline features/freaks/src/main/res/values-sw720dp/dimensions720dp.xml:16]: typo

File: features/freaks/src/main/res/values-sw720dp/dimensions720dp.xml [hunk 5]
```diff
@@ -13,4 +13,7 @@
...
+    <dimen name="image_roundend_background_size">250dp</dimen>
+    <dimen name="image_roundend_background_radius">15dp</dimen>
```
Comment (cipry13) [inline features/freaks/src/main/res/values-sw720dp/dimensions720dp.xml:18]: typo

File: features/freaks/src/main/kotlin/com/agilefreaks/freaks_catalog/features/freaks/details/FreakDetailsDataSource.kt [hunk 6]
```diff
@@ -7,11 +7,11 @@ import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.withContext
 
 interface FreakDetailsDataSource {
-    suspend fun getFreaks(): FreakDetailsQuery.Data?
+    suspend fun getFreaks(freakId: String): FreakDetailsQuery.Data?
```
Comment (AlexandraDamaschin) [inline features/freaks/src/main/kotlin/com/agilefreaks/freaks_catalog/features/freaks/details/FreakDetailsDataSource.kt]: Does this function really get Freaks?

File: features/freaks/src/main/res/layout/fragment_freak_details.xml [hunk 7]
```diff
@@ -44,21 +70,6 @@
...
-                android:elevation="@dimen/profile_photo_elevation"
-                android:scaleType="fitXY"
-                app:imageUrl="@{freakDetailsViewModel.freak.photo}"
-                app:layout_constraintDimensionRatio="1:1"
-                app:layout_constraintEnd_toEndOf="parent"
-                app:layout_constraintStart_toStartOf="parent"
-                app:layout_constraintTop_toTopOf="parent" />
-
             <androidx.constraintlayout.widget.Guideline
                 android:id="@+id/cardViewStartG"
```
Comment (AlexandraDamaschin) [inline features/freaks/src/main/res/layout/fragment_freak_details.xml]: What is cardViewStartG?

**Review by AlexandraDamaschin — CHANGES_REQUESTED** [review]
Some small improvements

**Review by AlexandraDamaschin — APPROVED** [review]
Does this improve software quality? Yes

**Pull request merged**